cloud/aws/modules/pgadmin/main.tf

```diff
@@ -26,7 +26,7 @@ resource "aws_lb_listener" "pgadmin" {
 
 # Traffic from load balancer is forwarded to IPs in this target group.
 resource "aws_lb_target_group" "pgadmin" {
-  name        = local.name_prefix
+  name        = substr(local.name_prefix, 0, 32)
```

Comment:
**suggestion:** Are there other resources that might run into this? Thoughts on updating the prefix local instead? And / or we can change it from "${var.app_prefix}-civiform-pgadmin" to "${var.app_prefix}-cf-pgadmin" to make this less likely to happen https://github.com/civiform/cloud-deploy-infra/blob/91ce382eef8f87804d72d8d1c0a74706b3b86aa5/cloud/aws/modules/pgadmin/main.tf#L99C17-L99C53

Reply:
oh I should add a comment explaining why